lib/chip-types/collection.js

```diff
@@ -72,7 +74,7 @@ Collection.prototype.get_unknown_field_errors = function(values){
 	const validation_errors = {};
 	for (const field_name in values){
 		if (this.fields[field_name] === undefined){
-			validation_errors[field_name] = new Errors.ValidationError(`Unknown field '${field_name}' for resource-type '${this.name}`);
+			validation_errors[field_name] = new Boom.badRequest(`Unknown field '${field_name}' for collection '${this.name}`);
```

Comment:
I think that this is a nice opportunity to refactor that piece of code. Here we're not creating a full user-facing error, message, but only a small part of it, that we're going to combine into one, large error in the line ~155. I think that this here could be just an object describing what the problem with the field is, not necessarily an object.

The above applies to all the `badRequest` errors in this file, except for the last one.